docs/generate-and-template.md

```diff
@@ -173,10 +173,11 @@ However, environment variables are expanded.
 Along with any existing environment variables, the following variables
 are specially defined:
 
-- HELM_HOME: The Helm home directory
-- HELM_DEFAULT_REPO: The repository alias for the default repository.
-- HELM_GENERATE_FILE: The present file's name
-- HELM_GENERATE_DIR: The absolute path to the chart directory of the present chart
+- `HELM_HOME`: The Helm Classic home directory
+    - Although the value of the `HELMC_HOME` environment variable, if defined, may play a role in defining the specially expanded `HELM_HOME` variable, the two are not equivalent. To produce charts that remain compatible with the _original_ Helm tool, which has now become Helm Classic, only the `HELM_HOME` variable should be used within `helm:generate` headers.
```

Comment:
I couldn't quite understand this part. I can't tell if or how `$HELMC_HOME` is used, and if `$HELM_HOME` is preferred for backward-compatibility, do we need to mention `$HELMC_HOME` at all?